### ISSUE TITLE
I'm getting the following console error when trying to delete a redirect

### ISSUE THREAD

**Issue by ddkpdev** [riasvdv/statamic-redirect#13]
I'm getting the following error message in the console when I try to delete a redirect.

`vue.common.prod.js:6 TypeError: Cannot read property '1' of undefined
    at a.deletingModalTitleFromRowKey (cp.js:1)
    at a.deletingModalTitle (cp.js:1)
    at fn.get (vue.common.prod.js:6)
    at fn.evaluate (vue.common.prod.js:6)
    at a.deletingModalTitle (vue.common.prod.js:6)
    at fn (cp.js:1)
    at r (vue.common.prod.js:6)
    at a.bt [as _t] (vue.common.prod.js:6)
    at Table.vue?3a47:5
    at a.vt [as _l] (vue.common.prod.js:6)
qe @ vue.common.prod.js:6
Be @ vue.common.prod.js:6
Re @ vue.common.prod.js:6
e._render @ vue.common.prod.js:6
r @ vue.common.prod.js:6
fn.get @ vue.common.prod.js:6
fn.run @ vue.common.prod.js:6
un @ vue.common.prod.js:6
(anonymous) @ vue.common.prod.js:6
Ge @ vue.common.prod.js:6
Promise.then (async)
Ue @ vue.common.prod.js:6
tt @ vue.common.prod.js:6
(anonymous) @ vue.common.prod.js:6
fn.update @ vue.common.prod.js:6
de.notify @ vue.common.prod.js:6
set @ vue.common.prod.js:6
hn.set @ vue.common.prod.js:6
confirmDeleteRow @ cp.js:1
click @ cp.js:1
$e @ vue.common.prod.js:6
n @ vue.common.prod.js:6
$e @ vue.common.prod.js:6
e.$emit @ vue.common.prod.js:6
selectAndClose @ DropdownItem.vue:33
$e @ vue.common.prod.js:6
n @ vue.common.prod.js:6
Kr.o._wrapper @ vue.common.prod.js:6
`

This effectively means, that it is impossible to delete records right now.

Using the latest version of Statamic: 3.1.14
Using version 1.4.6 of this plugin.

**Comment by jonatanalvarsson** [comment]
I'm having the exact same problem.

Statamic 3.1.23
Statamic Redirect 1.4.6

```
TypeError: Cannot read property '1' of undefined
    at a.deletingModalTitleFromRowKey (cp.js:1)
    at a.deletingModalTitle (cp.js:1)
    at fn.get (vue.common.prod.js:6)
    at fn.evaluate (vue.common.prod.js:6)
    at a.deletingModalTitle (vue.common.prod.js:6)
    at fn (cp.js:1)
    at r (vue.common.prod.js:6)
    at a.bt [as _t] (vue.common.prod.js:6)
    at Table.vue?56c8:5
    at a.vt [as _l] (vue.common.prod.js:6)
```

**Comment by JayGeorge** [comment]
I'm getting the same problem trying to delete a redirect but with this error in the console:

```
TypeError: this.rows is undefined
    deletingModalTitleFromRowKey https://nameofsite.wip/vendor/statamic-redirect/js/cp.js:1
    deletingModalTitle https://nameofsite.wip/vendor/statamic-redirect/js/cp.js:1
    get https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    evaluate https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    vn https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    fn https://nameofsite.wip/vendor/statamic-redirect/js/cp.js:1
    r https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    bt https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    o https://nameofsite.wip/vendor/statamic/cp/js/app.js?v=3.1.27:2
    vt https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    o https://nameofsite.wip/vendor/statamic/cp/js/app.js?v=3.1.27:2
    _render https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    r https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    get https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    run https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    un https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    tt https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    Ge https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    promise callback*Ue https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    tt https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    update https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    update https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    notify https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    set https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    set https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    confirmDeleteRow https://nameofsite.wip/vendor/statamic-redirect/js/cp.js:1
    click https://nameofsite.wip/vendor/statamic-redirect/js/cp.js:1
    $e https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    n https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    $e https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    $emit https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    selectAndClose https://nameofsite.wip/vendor/statamic/cp/js/app.js?v=3.1.27:2
    $e https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    n https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
    _wrapper https://nameofsite.wip/vendor/statamic/cp/js/vendor.js?v=3.1.27:2
```

**Comment by dragank35** [comment]
I am also getting an error when trying to delete a redirect. Stacktrace below:
`vue.common.prod.js:6 TypeError: Cannot read property '0' of undefined
    at a.deletingModalTitleFromRowKey (cp.js:1)
    at a.deletingModalTitle (cp.js:1)
    at fn.get (vue.common.prod.js:6)
    at fn.evaluate (vue.common.prod.js:6)
    at a.deletingModalTitle (vue.common.prod.js:6)
    at fn (cp.js:1)
    at r (vue.common.prod.js:6)
    at a.bt [as _t] (vue.common.prod.js:6)
    at Table.vue?b814:5
    at a.vt [as _l] (vue.common.prod.js:6)`

Statamic - v3.1.32
Statamic Redirect - v1.4.6

Does someone have a workaround for this issue?

**Comment by riasvdv** [comment]
This should be fixed in 1.4.7, make sure you republish the assets and you're getting the new cp.js from statamic-redirect
